packages/edit-site/src/components/sidebar-edit-mode/page-panels/index.js

```diff
@@ -75,7 +76,7 @@ export default function PagePanels() {
 							sprintf(
 								// translators: %s: Title of the created post e.g: "Post 1".
 								__( '"%s" successfully created.' ),
-								title
+								decodeEntities( title )
```

Comment:
@jorgefilipecosta Hi, where in the interface is this "Sidebar edit mode page panel" ? I am not sure how to test this, but I could tell the title was not already decoded.